CONTRIBUTORS.md

```diff
@@ -45,3 +45,4 @@
 * Tim Clark
 * Tobias Nießen
 * Vanh Khuyen Nguyen
+* Vanh Nguyen
```

Comment:
Weird that npm generated another entry for Vanh. Not sure if this can be corrected for next time?

Reply:
I'll leave it for now (CI is running) but I'd have to look at the script to see how that's pulled. Curious if maybe a commit was associated with a different email address (I've done that occasionally if Github decides to use my personal vs work email) and therefore a different name?